packages/material-ui-styled-engine-sc/src/index.js

```diff
@@ -1,14 +1,18 @@
 import scStyled from 'styled-components';
 
 export default function styled(tag, options) {
+  let scStyledPrepared = scStyled(tag);
+
   if (options) {
-    return scStyled(tag).withConfig({
+    scStyledPrepared = scStyled(tag).withConfig({
       displayName: options.label,
       shouldForwardProp: options.shouldForwardProp,
     });
   }
 
-  return scStyled(tag);
+  return (styles) => {
```

Comment:
Necessary for fixing `You have illegal escape sequence in your template literal`

Reply:
Should we leave a comment in this case?

Reply:
We should have a better fix for this in my opinion, so I left TODO for it. I will start with that after I am done with the system